### PR TITLE
add badge

### PR DESCRIPTION
**Pull request by sirenkovladd**
add badges to README

### CODE REVIEW

**Comment by Uzlopak** [comment]
Whats benefit of the release badge?

**Comment by sirenkovladd** [comment]
better readme
link to npm
developer friendly)

**Comment by Uzlopak** [comment]
Do you know what the Release badge actually means?

**Comment by sirenkovladd** [comment]
Was the last action successful?

**Comment by Uzlopak** [comment]
yeah but did you ever check what the release.yml does?

**Comment by sirenkovladd** [comment]
Release to github :smile: 
I can change to Test CI

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # tinybench
 
+[![Test](https://github.com/tinylibs/tinybench/actions/workflows/test.yml/badge.svg?branch=main)](https://github.com/tinylibs/tinybench/actions/workflows/test.yml)
```
Comment (Uzlopak) [inline README.md]:
```suggestion
[![CI](https://github.com/tinylibs/tinybench/actions/workflows/test.yml/badge.svg?branch=main)](https://github.com/tinylibs/tinybench/actions/workflows/test.yml)
```

**Comment by sirenkovladd** [comment]
@Uzlopak

**Comment by Uzlopak** [comment]
I am knee deep in other things currently.

**Review by Uzlopak — APPROVED** [review]
LGTM

**Pull request merged**